### DISCUSSION TITLE
Termux download

### DISCUSSION THREAD

**Discussion by Crashito67** [community/community#53639] [Programming Help]
Where do I find the list of downloads for termux arm 64 v8 118.0 app

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Crashito67 ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
